src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json

```diff
@@ -0,0 +1,156 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.DataProtection.AzureStorage, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
To confirm: This file was generated after building this project in the 1.0.0 branch or tag?